### ISSUE TITLE
Ponomar Unicode should support more archaic letters

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [typiconman/fonts-cu#4]
This font should include more archaic letters, especially from Cyrillic Extended-B block, they are also had historical usage. As my observe, the following letters can get support firstly:
Dzělo, U+A642—A643
Djerv, the origin of Ћ and Ђ, U+A648—A649
Neutral Yer, U+A64E—A64F
An alternate version of Yery (with “back Yer”), U+A650—A651
Iotified Yat, U+A652—A653
Reversed Yu, U+A654—A655
Kopa, U+0480—0461

**Comment by starover77** [comment]
The font "Ponomar Unicode" does not need to support more archaic characters because such characters are not needed for Church Slavonic typography. The Ponomar Project provides other fonts which are appropriate for the characters you suggest.

As a principle, the Ponomar Project tries to provide fonts which conform to specific genres or time periods of Slavonic literature, including: Ustav, Poluustav, Synodal Era Church Slavonic, Skoropis, etc. If you wish to typeset texts using the archaic characters you suggest, then you need to choose an appropriate archaic (Ustav era) font. Using Ponomar Unicode to typeset Paleoslavic texts is like using Old English lettering to advertise a concert of rock music.

As a specific example of usage which conforms to "period-appropriate" typography, please look at the Slavonic dictionary of D'iachenko; he used specific typefaces which are appropriate for Paleoslavic, Church Slavonic and Old Russian. We encourage you to follow this example if you wish to work with archaic texts. It is a bad habit to use fonts in an anachronistic manner, and if you are typesetting academic or professional literature, your peers would expect you to use a "period-appropriate" style of font.

**Comment by typiconman** [comment]
Yes, support of a archaic letters is provided in the Mention Unicode font. For a Synodal error font this is anachronistic.

**Comment by KrasnayaPloshchad** [comment]
OK, I understand since I got UTN No.41. I was got inspired by English Wikipedia “[Early Cyrillic alphabet](https://en.wikipedia.org/wiki/Early_Cyrillic_alphabet)” before I report, the alphabet table seems in ustav, but the images looks like synodal styled face.

**Comment by starover77** [comment]
I'm sorry, but Wikipedia is not an academically accepted source of reliable information.

We have discussed this topic in the past, and we decided that further discussion is closed. Our decision was: Archaic Cyrillic letters will be added to Ustav style fonts, but they will not be added to Synodal era fonts because to do this would create an anachronism and will allow uneducated users to use (i.e. misuse, abuse) the fonts for purposes that are contrary to historical accuracy. Sorry, but this subject remains closed.
